proto/storage/compliance_operator_v2.proto

```diff
@@ -35,7 +35,8 @@ enum RuleSeverity {
 }
 
 message ProfileShim {
-  string profile_id = 1; // @gotags: search:"Profile Ref ID,hidden" sql:"fk(ComplianceOperatorProfileV2:profile_ref_id),no-fk-constraint,type(uuid)"
+  string profile_id = 1;
```

Comment:
This change is needed because in 4.4 the profile_id was defined as a string not a UUID and mapped to the `profile_id` in the Profile.  When I introduced the `profile_ref_id` in 4.5 I simply replaced the `profile_id`.  I should not have because that changes the type.  So the proper solution is to carry both fields.  So that is what this change does.  The only place this will cause an issue is staging and that can be resolved by cleaning up the data.